ts/util/attachmentsUtil.ts

```diff
@@ -53,7 +54,7 @@ export async function autoScaleForAvatar<T extends { contentType: string; blob:
 ) {
   const maxMeasurements = {
     maxSide: AVATAR_MAX_SIDE,
-    maxSize: 1000 * 1024,
+    maxSize: 5 * 1024 * 1024,
```

Comment:
Might be nice to have a constant for this. See usage of `DURATIONS` as an example.

```tsx
export const FILESIZES = {
  /** 1KB */
  KB: 1024,
  /** 1MB */
  MB: 1024 * 1024,
  /** 1GB */
  GB: 1024 * 1024 * 1024,
};

You can choose a better name if you can think of one
```

Reply:
and replace any other file sizes in the project please

Reply:
I'd argue that a constant used only in one file should stay in that file (but still a const at the top)